src/app/api/question/submit/route.ts

```diff
@@ -76,6 +77,19 @@ async function sendQuestion(
 export async function POST(request: NextRequest) {
   const res = await request.json()
 
+  const isQuestionContainsBadWord =
+    res.question
+      .split(' ')
+      .filter((word: string) => BAD_WORDS.includes(word.toLowerCase())).length >
```

Comment:
Prefer using key-value data stucture, to reduce the time complexity.

https://www.tech-hour.com/javascript-performance-and-optimization